gym/wrappers/step_api_compatibility.py

```diff
@@ -15,33 +18,33 @@ class StepAPICompatibility(gym.Wrapper):
 
     Args:
         env (gym.Env): the env to wrap. Can be in old or new API
-        new_step_api (bool): True to use env with new step API, False to use env with old step API. (False by default)
+        to_termination_truncation_api (bool): True to use env with new step API, False to use env with old step API. (False by default)
```

Comment:
I think it's better to use `to_terminated_truncated_api` to be consistent with the actual variable names